tst/testinstall/strings.tst

```diff
@@ -262,6 +262,14 @@ gap> fname := Filename(tmpdir, "output.csv");;
 gap> PrintCSV( fname, s );
 gap> s = ReadCSV( fname );
 true
+gap> s[2].emil:="\"Alas, poor Yorick\", the call went";;
+gap> PrintCSV(fname,s);
+gap> ReadCSV(fname,true);
+[ rec( field1 := "emil", field2 := "f", field3 := "f1", field4 := "f 2" ), 
+  rec( field2 := 2, field3 := 1, field4 := 3 ), 
+  rec( field1 := "\"Alas, poor Yorick\", the call went", field3 := 4, 
+      field4 := 6 ), rec( field3 := 7 ), rec( field2 := 11, field4 := 12 ), 
+  rec( field2 := "yy", field3 := "x", field4 := "zzz" ) ]
```

Comment:
Can you please also add an "empty" field? I.e. containing an empty string.

Reply:
Done